Oqtane.Server/Controllers/JobController.cs

```diff
@@ -34,6 +36,18 @@ public IEnumerable<Job> Get()
             return _jobs.GetJobs();
         }
 
+        // GET: api/<controller>
+        [HttpGet("list")]
+        [Authorize(Roles = RoleNames.Host)]
+        public IEnumerable<string> List()
+        {
+            var types = AppDomain.CurrentDomain
```

Comment:
Can we introduce a `static` method that retrieved a hosted services, because this may an expenssive operation

Reply:
Not critical operation, how often this api will be called? No reason to do it. You can cache results somewhere, but why?

Reply:
> how often this api will be called? 

I expect every time the jobs page is called

Reply:
@chlupac  is correct - the Add Job UI is very rarely called so this list does not need to be cached.

Reply:
If is that no need to change this

Reply:
@hishamco if we wanted to optimize for performance, the best place to introduce a static would in the GetOqtaneAssemblies extension method, as this is called repeatedly during startup and during other operations.